### ISSUE TITLE
Import/export issues

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#6518]
### Observed behavior

Observed by a [user in the forums](https://community.learningequality.org/t/import-export-issues/1396):

> I am experiencing al multitude of issues with Kolibri.
> 
>1. When I import several resources at the same time the Task Manager gets stuck at 100%. Cancelling the import doesn’t work. Only after restarting the computer does the task disappear. But afterwards I get the same problem with every task. So completely unable to import anything. I have to uninstall (deleting all imports) and reinstall Kolibri to be able to import again.
>2. When importing resources (Khan Academy) via a local drive I get the error message “Channel not found” even though Kolibri recognizes the exported resources on the drive.
>3. Kolibri often refuses to start up. Again uninstall - reinstall seems to be the only way to solve this.

### Expected behavior

import runs correctly

### User-facing consequences

cannot get content

### Errors and logs

```
ERROR 2020-02-01 13:32:26,810 exception Internal Server Error: /api/content/contentnode_granular/1ceff53605e55bef987d88e0908658c5/
Traceback (most recent call last):
File “C:\Python34\lib\site-packages\kolibri\dist\sqlalchemy\engine\base.py”, line 1193, in _execute_context
context)
File “C:\Python34\lib\site-packages\kolibri\dist\sqlalchemy\engine\default.py”, line 509, in do_execute
cursor.execute(statement, parameters)
sqlite3.OperationalError: too many SQL variables
```

### Steps to reproduce

Unknown

### Context

Kolibri 0.13.0
Windows 10 Home
Windows Explorer

**Comment by indirectlylit** [comment]
might be related to https://github.com/learningequality/kolibri/issues/6157 and https://github.com/learningequality/kolibri/pull/6378

**Comment by lyw07** [comment]
fixed in #6527
